src/redis-cli.c

```diff
@@ -261,6 +263,7 @@ static struct config {
     int resp3; /* value of 1: specified explicitly, value of 2: implicit like --json option */
     int in_multi;
     int pre_multi_dbnum;
+    int json_binary_encoding;
```

Comment:
This fields isn't initialized. 
Is it expected to be one of JSON_ENABLE_BINARY_ENCODING / JSON_DISABLE_BINARY_ENCODING? 
I.e. 1 or 2?
Why not make it a boolean?

Reply:
I thought that other options like BASE64 format, but I don't think I want to implement them in this time because I'm not sure how much the feature will be useful currently, then I'll fix it as boolean.

Reply:
don't forget to init it in the beginning of `main()`